pkg/handlers/cni/calico/manifests/docker/installation.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# This section includes base Calico installation configuration.
+# For more information, see: https://docs.projectcalico.org/reference/installation/api
+apiVersion: operator.tigera.io/v1
+kind: Installation
+metadata:
+  name: default
+spec:
+  cni:
+    type: Calico
+  # Configures Calico networking.
+  calicoNetwork:
+    # Note: The ipPools section cannot be modified post-install.
+    ipPools:
+    - blockSize: 26
+      cidr: ${POD_SUBNET}
```

Comment:
Should `cidr` be set to an empty value or a default value instead? If I understand this not a used as a template right?

Reply:
Yes, this should just contain the default... I got caught up in trying to use unstructured helpers to set a field in an element in a nested slice 😅. I'll add the default directly in here and then figure out how we can allow overrides. I'm starting to think about discover variables mutation hook to be able to specify it rather than annotations, but that will be more work so will start with annotations.

Reply:
Added default into file, and allowed override via annotation in latest commit.